accounts/wallet_account.go

```diff
@@ -0,0 +1,32 @@
+package accounts
+
+import (
+	"errors"
+)
+
+// WalletAccount wraps wallet with a "default" account
+type WalletAccount interface {
+	Wallet
+	Account() Account
+}
+
+type walletAccount struct {
+	Wallet
+	account Account
+}
+
+var (
+	errInvalidAccountAddress = errors.New("invalid account address, doesnt exists in wallet")
```

Comment:
we should move this one to the beginning of the file - they follow that convention in go-eth afaik

Reply:
also I think that it would be important to show the the account address that is invalid.

Reply:
:+1: 
will make the changes